Module.yaml

```diff
@@ -0,0 +1,62 @@
+version: 1.0.0
+name: OPCUADiscovery
+description: OPC UA Discovery looks for all available nodes onn OPC UA Server and forwards the payload to next module.
+icon: https://icons-020-demo.s3.eu-central-1.amazonaws.com/mi_OPCUA_input.png
```

Comment:
Same as in another module, it is `process` type module and you use `input` icon. Please ask Christina for process module icon for OPCUA

Reply:
we should use same icon for all these opc ua modules, kinda makes no sense anyway to have these names for icons anyway ? I'll request new icon, but I think I'll get the same.

Reply:
module is changed to input based on our last call.